extension/js/common/platform/debug.ts

```diff
@@ -0,0 +1,51 @@
+/* ©️ 2016 - present FlowCrypt a.s. Limitations apply. Contact human@flowcrypt.com */
+
+'use strict';
+
```

Comment:
Please add a comment here about what this can be used for and how - eg explain how this can be used to extract input/output of methods during tests. Then also add a note in repo readme to point to this file/comment as well. Thanks!

Reply:
Done